test/wpt/runner/runner.mjs

```diff
@@ -307,16 +322,23 @@ export class WPTRunner extends EventEmitter {
     }
 
     this.emit('completion')
-    const { completed, failed, success, expectedFailures, skipped } = this.#stats
+
+    const { testsPassed, testsFailed, testsSkipped } = this.#stats
+    console.log(
+      `Test    results for folder [${this.#folderName}]: ` +
```

Comment:
`Test` should be replaced with `File`, as this seems to be the number of files run.